Backpack/Calendar/Classes/BPKCalendarAppearance.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@interface BPKCalendarAppearance : NSObject
```

Comment:
Do we really need to be able to customize the appearance this much? Couldn't we make the first version in a single style and then add something like this if required?

Reply:
I am more than happy to cut scope. We could remove the whole BPKCalendarAppearance.h altogether.